service_thing_aggregate/app/repositories/dao/ObjectDao.scala

```diff
@@ -352,13 +350,12 @@ class ObjectDao @Inject() (
         WHERE lo."MUSEUM_ID" = ${mid.underlying}
         AND lo."CURRENT_LOCATION_ID" = ${nodeId.underlying}
         AND mt."OBJECT_ID" = lo."OBJECT_ID"
-        AND mt."IS_DELETED" = 0
-        #${collectionFilter(collections)}
+        AND mt."IS_DELETED" = 0 #${collectionFilter(collections)}
```

Comment:
Placing the `#{...` line after the `"IS_DELETED"` clause, renders the query a bit more visually pleasing when `DEBUG` logging is enabled for slick.